dsgrid/utils/spark.py

```diff
@@ -129,20 +129,28 @@ def _post_process_dataframe(df, cache=False, require_unique=None):
                     raise DSGInvalidField(f"DataFrame has duplicate entries for {column}")
 
 
-def get_unique_values(df, column):
-    """Return the unique values of a dataframe in one column.
+def get_unique_values(df, columns: [str, list]):
+    """Return the unique values of a dataframe in one column or a list of columns.
 
     Parameters
     ----------
     df : pyspark.sql.DataFrame
-    column : str
+    column : str or list of str
 
     Returns
     -------
     set
 
     """
-    return {getattr(x, column) for x in df.select(column).distinct().collect()}
+    dfc = df.select(columns).distinct().collect()
+    if type(columns) == list:
+        values = set()
+        for row in dfc:
```

Comment:
Technically, it's more performant to make it a set comprehension. Since this could be large and it's easy to do here, I would.

Reply:
I am following the output format from `itertools.product(*dim_records_list)`, which gives a list/set of tuples. Unless I write a custom func to replace itertools.product()?

Reply:
I just meant like this:
```
values = {tuple(getattr(row, col) for col in columns) for row in dfc}
```